doc/api/http.md

```diff
@@ -56,13 +56,13 @@ The HTTP Agent also defaults client requests to using
 socket to become free the socket is closed. This means that Node.js's
 pool has the benefit of keep-alive when under load but still does not
 require developers to manually close the HTTP clients using
-KeepAlive.
+keepAlive.
 
-If you opt into using HTTP KeepAlive, you can create an Agent object
+If you opt into using HTTP keepAlive, you can create an Agent object
```

Comment:
I guess a follow on question would be whether we should always scope the object name, so that whenever `Agent` appears in backticks, it is written as `http.Agent`.